pkg/cmd/admin/policy/reconcile_clusterroles.go

```diff
@@ -148,9 +155,60 @@ func (o *reconcileClusterOptions) ChangedClusterRoles() ([]*authorizationapi.Clu
 		}
 
 		if !kapi.Semantic.DeepEqual(expectedClusterRole.Rules, actualClusterRole.Rules) {
+			if o.Union {
+				modified := []authorizationapi.PolicyRule{}
```

Comment:
This name isn't quite right.  This is the list of rules with no equivalent rule in the existing role, right?  MissingRules?